main/swadge_esp32.c

```diff
@@ -323,7 +323,8 @@ void mainSwadgeTask(void* arg __attribute((unused)))
         {
 #if !defined(CONFIG_SWADGE_DEVKIT)
             // If test mode was passed
-            if(getTestModePassed())
+//            if(getTestModePassed())
```

Comment:
The devkits should pass the test program. Please undo this

Reply:
OH CRUD WHOOPS  WILL FIX NOW.

Reply:
fixed.